App.js

```diff
@@ -236,13 +635,18 @@ export default function App() {
   return (
     <View style={styles.container}>
       <View style={styles.nav}>
-        <Button onPress={() => pressReset()} title="Reset" color="blue" />
+        <Button
+          onPress={() => pressReset()}
+          title="Reset"
+          color="blue"
+          disabled={numberOfPlayers === 0 ? true : false}
+        />
         <View
           style={
-            endOfGame
+            endOfGame || numberOfPlayers === 0
```

Comment:
How could numberOfPlayers ever be 0?!?

Reply:
A value of of '0' indicated that no selection if the game is a 1 player or 2 player game has not occurred.